state/address_test.go

```diff
@@ -209,54 +209,6 @@ func (s *ControllerAddressesSuite) TestSetAPIHostPortsNoMgmtSpaceConcurrentDiffe
 	c.Assert(hostPorts, gc.DeepEquals, []network.SpaceHostPorts{hostPorts1})
 }
 
-func (s *ControllerAddressesSuite) TestSetAPIHostPortsWithMgmtSpace(c *gc.C) {
```

Comment:
We need to be careful to remember this coverage. When we convert link-layer devices and implement agent config updates based off data in Dqlite.